FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - [changed] Functionally neutral change to stop using a deprecated method in the AB Testing API. (#6543)
 - [fixed] Updated `numberValue` to be nonnull to align with current behavior. (#6623)
 - [removed] Removed deprecated APIs `isDeveloperModeEnabled`, `initWithDeveloperModeEnabled:developerModeEnabled`, `activateWithCompletionHandler:completionHandler`, `activateFetched`, `configValueForKey:namespace`, `configValueForKey:namespace:source`, `allKeysFromSource:namespace`, `keysWithPrefix:namespace`, `setDefaults:namespace`, `setDefaultsFromPlistFileName:namespace`, `defaultValueForKey:namespace`. (#6637)
+- [fixed] Completion handler for `fetchAndActivateWithCompletionHandler` is now run on the main thread. (#6665)
```

Comment:
Prefer issue number over PR for Changelogs.

Reply:
I see, updated!